server/@types/express/index.d.ts

```diff
@@ -21,5 +22,9 @@ export declare global {
       id: string
       logout(done: (err: unknown) => void): void
     }
+
+    interface Locals {
+      user: Express.User
+    }
```

Comment:
The `setUpCurrentUser` middleware always populates `res.locals.user` via the user service